pyproject.toml

```diff
@@ -0,0 +1,34 @@
+[build-system]
+requires = ["setuptools>=64", "setuptools-scm"] 
```

Comment:
Installing a plain Python package is such a trivial task that I don't like to to add extra requirements just for this. setuptools (without version constraint) is an exception that I'm willing to take, but I prefer not to go beyond that.